ajax/libs/nipplejs/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "nipplejs",
+  "version": "0.6.7",
+  "filename": "nipplejs.min.js",
+  "description": "A virtual joystick for touch capable interfaces",
+  "author": "Yoann Moinet <yoann.moinet@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yoannmoinet/nipplejs.git"
+  },
+  "homepage": "http://yoannmoinet.github.io/nipplejs",
```

Comment:
Let's use `https`
Using https is better because it's more security for browsing.